server/src/useCases/report/productTestCases/handleGetProductDetailsRequest.ts

```diff
@@ -1,26 +1,39 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable no-param-reassign */
 /* eslint-disable no-console */
-const {
-  mapQueryToSorting,
-} = require('../requestUtils');
+import { Request, Response } from 'express';
+import mapQueryToSorting from '../requestUtils';
 
-module.exports = class ReportGetProductDetailsRequestHandler {
-  constructor(request, response) {
+interface Repository {
+  aggregateBBDetailsByProductId(params: {
+    id: string;
+  }, sorting: any, callback: (err: ErrorType, result: any) => void): void;
+}
+
+export default class ReportGetProductDetailsRequestHandler {
+
+  public req: Request;
+  public res: Response;
+  public dbConnect: any;
```

Comment:
Please use correct dbConnect type.

Reply:
I will add collection type to this one: Model<Document>